cmd/util/cmd/epochs/cmd/recover.go

```diff
@@ -44,29 +43,19 @@ func init() {
 }
 
 func addGenerateRecoverEpochTxArgsCmdFlags() {
-	generateRecoverEpochTxArgsCmd.Flags().StringVar(&flagBucketNetworkName, "bucket-network-name", "",
-		"when retrieving the root snapshot from a GCP bucket, the network name portion of the URL (eg. \"mainnet-13\")")
-	generateRecoverEpochTxArgsCmd.Flags().IntVar(&flagCollectionClusters, "collection-clusters", 0,
+	generateRecoverEpochTxArgsCmd.Flags().IntVar(&flagCollectionClusters, "collection-clusters", 3,
 		"number of collection clusters")
 	// required parameters for network configuration and generation of root node identities
-	generateRecoverEpochTxArgsCmd.Flags().StringVar(&flagConfig, "config", "",
+	generateRecoverEpochTxArgsCmd.Flags().StringVar(&flagNodeConfigJson, "node-config", "",
```

Comment:
```suggestion
	generateRecoverEpochTxArgsCmd.Flags().StringVar(&flagNodeConfigJson, "config", "",
```
I think this flag has the same meaning as other commands where the flag name `--config` is used. Although I agree `node-config` is more descriptive, I'd suggest to use the same flag name as the other commands for consistency.